### PR TITLE
Harmonize admontions in style guide

### PR DESCRIPTION
**Pull request by malfisya**
## Description

<Describe what this pull request adds>

This PR aims to harmonize our admonitions style guide to what is available on Docusaurus. The wording are adapted from [Grafana](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/) and [Codeberg](https://docs.codeberg.org/improving-documentation/style-guide/) style guides. Also with a little bit help of ChatGPT. The prompt is attached below :

> Me : We want to harmonize our style guide on admonitions with whats available on docusaurus.
What we have right now are adopted from Redhat. Redhat have four admonitions: Note, Important, Warning/Danger, and Tip. Docusaurus gives us five options : Note, Tip, Info, Important, Warning and Danger. Notice how docusaurus separate Warning and Danger, while Redhat don't. How would you describe those admonitions?
>
> ChatGPT : Sure, here's a breakdown of how you might describe each of these admonitions:
>1. **Note**: Used to provide additional information or clarification that is not critical but can enhance understanding.
>2. **Tip**: Offers a helpful suggestion or advice that can improve the user's experience or workflow.
>3. **Info**: Provides general information that may be useful or relevant to the reader but isn't necessarily a warning or a tip.
>4. **Important**: Highlights information that is crucial for the reader to understand or follow for successful completion of a task or to avoid issues.
>5. **Warning**: Indicates a potential hazard, risk, or consequence that the reader should be aware of and take precautions against.
>6. **Danger**: Indicates an imminent or severe risk that could lead to serious consequences if not addressed or avoided immediately.
>While Redhat combines "Warning" and "Danger" into one category, Docusaurus separates them, perhaps to provide more granularity in communicating the severity of the cautionary message.

Resolves getsolus/help-center-docs#495

### CODE REVIEW

File: docs/user/contributing/style.md [hunk 1]
```diff
@@ -34,16 +34,19 @@ _In alphabetical order, not order of importance._
...
 | Note           | Additional guidance or advice that improves product configuration, performance, or supportability                                                                                                                                    |
-| Important      | Advisory information essential to the completion of a task or default configuration settings. Users must not disregard this information.                                                                                             |
-| Warning/Danger | Information about potential system damage, data loss, or a support-related issue if the user disregards this admonition. If available, offer information to avoid the problem in the future or state where to find more information. |
 | Tip            | Recommendations, suggestions, and alternative methods that might not be obvious. Tips are not essential to using the product.                                                                                                        |
+| Info           | General information about product that may be useful or relevant to user.           |
```
Comment (davidjharder) [inline docs/user/contributing/style.md:45]:
```suggestion
| Info           | General information about the topic at hand that may be useful or relevant to user.           |                                                                                                 
```

**Review by davidjharder — CHANGES_REQUESTED** [review]
one nit and a general note.

In future, let's be cautious or using ChatGPT to do the actual writing; although I can see you didn't actually use the suggestions directly here. We are asserting copyright on our help center content, and ChatGPT may muddy that.

File: docs/user/contributing/style.md (hunk 1 above)
Comment (malfisya) [reply docs/user/contributing/style.md:45]: Ok, will fix this tomorrow. Already winding down for today.

Comment (davidjharder) [reply docs/user/contributing/style.md:45]: You mind If I commit it myself then?

Comment (malfisya) [reply docs/user/contributing/style.md:45]: No, feel free to do so.

**Review by davidjharder — APPROVED** [review]
Thanks for the fix!

**Pull request merged**